test/test_tree_2d_linearizedeuler.jl

```diff
@@ -0,0 +1,21 @@
+module TestExamples2DEulerLinearized
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+EXAMPLES_DIR = joinpath(examples_dir(), "tree_2d_dgsem")
+
+@testset "Linearized Euler Equations 2D" begin
+  @trixi_testset "elixir_linearized_euler_convergence.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_linearized_euler_convergence_test.jl"),
+                        initial_refinement_level=4,
+                        tspan1=(0.0, 0.2), tspan_averaging=(0.0, 0.2), tspan=(0.0, 0.2),
```

Comment:
```suggestion
```
I don't think these lines are necessary, since they just repeat what is used in the elixir anyways (and `tspan1` and `tspan_averaging` are not used at all)